recipes/sides/freaky-freekeh.md

```diff
@@ -0,0 +1,33 @@
+# Freaky Freekeh
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-20%25-green.svg) 
+![Speed Badge](https://img.shields.io/badge/Speed-0.3hrs-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Cheap-green.svg)
+![Serves Badge](https://img.shields.io/badge/Serves-8-green.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Green Badge Count Badge](https://img.shields.io/badge/Green%20badge%20count-100%25-green.svg)
+
++ **Chef:** Suha
++ **Sous:** Lubes
++ **Date:** Week 12 FACN2
```

Comment:
wait what ?

Reply:
when we had the barbecue on the balcony, remember?

Reply:
I feel old :cry:

Reply:
back in those wild teenage days...